.github/workflows/node.js.yml

```diff
@@ -0,0 +1,28 @@
+name: Node.js CI
+
+on:
+  workflow_dispatch:
+  push:
+
+env:
+  CONTENTFUL_ACCESS_TOKEN: ${{ secrets.CONTENTFUL_ACCESS_TOKEN }}
+  CONTENTFUL_SPACE: ${{ vars.CONTENTFUL_SPACE }}
+  CONTENTFUL_ENVIRONMENT: ${{ vars.CONTENTFUL_ENVIRONMENT }}
+  CONTENTFUL_ENTRY: ${{ vars.CONTENTFUL_ENTRY }}
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: ['18.x', '20.x']
+
+    steps:
+      - uses: actions/checkout@v4
```

Comment:
As for every Open Source project under OB organization, we want to have harden runner and best security practices in our github actions. 

You can take a look at how to add harden runner [here](https://github.com/onebeyond/reusable-workflows/blob/3d2d8238cf3ffb9a8287bdbbd6454cdd1136f458/.github/workflows/openssf-scorecard.yml#L26)

And how to make reference to a specific commit of the actions [here](https://github.com/onebeyond/reusable-workflows/blob/3d2d8238cf3ffb9a8287bdbbd6454cdd1136f458/.github/workflows/openssf-scorecard.yml#L42)